src/index.ts

```diff
@@ -88,6 +94,10 @@ export async function resolveSchema (config: Config): Promise<Schema> {
               })
           }
         })
+     case 'schema':
+       const schemaSource = require(resolve(config.file))
```

Comment:
Ok some examples:

Given the cwd of `/var/app`

"file": "/schema.js"
* join: "/var/app/schema.js"
* resolve: "/schema.js"

"file": "./schema.js"
* join: "/var/app/schema.js"
* resolve: "/var/app/schema.js"

"file": "../schema.js"
* join: "/var/schema.js"
* resolve: "/var/schema.js"


"file": "schema.js"
* join: "/var/app/schema.js"
* resolve: "/var/app/schema.js"

The only difference is in the one with a leading slash.